File: pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int n = 0;
    int blanck = 0;
    int midle = 0;
    int hash = 0;

    do
    {
        n = get_int("How tall is the pyramid?\n");

    } while(n < 1 || n > 8);


    for(int i=0;i<n;i++)
    {
        for(int j = 0; j<n-i-1; j++)
        {
            printf(" ");
        }

        for(int j=0; j<i+1;j++)
        {
            printf("#");
        }

        for(int j=0; j<2;j++)
        {
            printf(" ");
        }

        for(int j=0; j<i+1;j++)
        {
            printf("#");
        }
        printf("\n");
    }



}

File: pset6/mario/more/mario.py
from  cs50 import get_int


def main():
    while True:
        h = get_int("Height: ")
        if h > 0 and h <= 8 :
            break;
    print_mario(h, h)

def print_mario(n, h):
    if n > 0:
        print_mario(n-1, h)
        print(f"{' ' * (h-n)}{'#' * n}  {'#' * n}")
main()